src/main/java/by/custom/utilcalculator/domain/tree/HelperTree.java

```diff
@@ -31,39 +36,35 @@ public static Map<StepsIndicator, String> fillFieldsToCommandsMap() {
     }
 
     public static Node buildTree() {
-        Node treeRoot = new Node(null, Command.START);
-        //eaes nodes queue
-        Node eaes = new Node(treeRoot, Command.EAES);
-        Node eaesPhysical = new Node(eaes, Command.PHYSICAL_PERSON);
-        Node eaesJuridical = new Node(eaes, Command.JURIDICAL_PERSON);
-        eaes.addChildren(Arrays.asList(eaesPhysical, eaesJuridical));
-        Node eaesPhysicalAge = new Node(eaesPhysical, Command.AGE);
-        eaesPhysical.addChildren(List.of(eaesPhysicalAge));
-        Node eaesJuridicalAge = new Node(eaesJuridical, Command.AGE);
-        eaesJuridical.addChildren(List.of(eaesJuridicalAge));
+        URL res = HelperTree.class.getClassLoader().getResource("tree.json");
+        File file = null;
+        try {
+            file = Paths.get(res.toURI()).toFile();
+        } catch (URISyntaxException e) {
+            e.printStackTrace();
+        }
 
-        //other physical nodes queue
-        Node other = new Node(treeRoot, Command.OTHER_COUNTRIES);
-        treeRoot.addChildren(Arrays.asList(eaes, other));
-        Node otherPhysical = new Node(other, Command.PHYSICAL_PERSON);
-        Node otherPhysicalAge = new Node(otherPhysical, Command.AGE);
-        otherPhysical.addChildren(List.of(otherPhysicalAge));
+        final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
It could be a static class field. You haven't to create it on every method call.